modules/ossm-excluding-individual-pods-from-cluster-wide-mesh-cli.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-excluding-individual-pods-from-cluster-wide-mesh-cli_{context}"]
+= Excluding individual pods from a cluster-wide mesh using the CLI
+
+A pod receives sidecar injection if it has the `sidecar.istio.io/inject: true` annotation applied, and the pod exists in a namespace that matches either the label selector or the members list defined in the `ServiceMeshMemberRoll` resource. 
+
+If a pod does not have the `sidecar.istio.io/inject` annotation applied, it cannot receive sidecar injection.
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator.
+* You have deployed a `ServiceMeshControlPlane` resource with the `mode: ClusterWide` annotation.
+* You are logged in as a user with the `cluster-admin` role. If you use {product-dedicated}, you are logged in as a user with the `dedicated-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} CLI.
+
+. Edit the deployment configuration.
++
+[source,terminal]
+----
+$ oc edit deployment -n <namespace> <deploymentName>
+----
+
+. Modify the YAML file to deploy one application that receives sidecar injection and one that does not.
```

Comment:
```suggestion
. Modify the YAML file to deploy one application that receives sidecar injection and one that does not, as shown in the following example:
```

Reply:
I think that the OpenShift doc guidelines to introduce an example (along with the guideline to introduce a command) have something to do with it making it easier for localization.